tests/specs/spa/capture-interactions.e2e.js

```diff
@@ -1,5 +1,7 @@
+import { notIE } from '../../../tools/browser-matcher/common-matchers.mjs'
+
 describe('spa captures interaction when', () => {
-  it('hashchange fires after finish', async () => {
+  it('hashchange fires after XHR loads', async () => {
```

Comment:
I know you only changed the name here but this test is sort of unnecessary and somehow expansive in what its testing at the same time. I don't think we need the XHR part of this test and could split the remaining into two parts: testing the interaction api and simply testing that the spa feature captures an interaction when the hash changes (separate from anything else).

Reply:
Group decided to keep these tests as 1:1 for conversion purposes and revisit usefulness another time